### PR TITLE
#1951 library loader fixes, welcome screens

### PR DESCRIPTION
**Pull request by finnfiddle**
- [ ] **Update release notes if necessary**

I added a welcome screen:

TODO: translations

Screenshot 2019-05-02 at 14 14 37

### CODE REVIEW

File: client/src/components/library/LibraryListing.jsx [hunk 1]
```diff
@@ -107,33 +108,45 @@ export default function LibraryListing({
...
   return (
-    <div className={`LibraryListing ${displayMode}`}>
-      {(!results && !currentCollection && !searchString) &&
-        <span
-          className="noItemsMessage"
+    <div className={`LibraryListing ${displayMode} ${!results ? 'LibraryListing--empty' : ''}`}>
+      {(!results && !currentCollection && !searchString) && (
+        <ErrorScreen
+          code="Welcome"
+          title="Get started by adding datasets, visualisations and dashboards to your library from the top menu."
```
Comment (tangrammer) [inline client/src/components/library/LibraryListing.jsx]: could we internationalize the message? :)

**Comment by janagombitova** [comment]
Looks wonderful @finnfiddle One tiny detail: the 'a' is not correctly grammatically. So it should be "Get started by adding datasets... "

**Comment by finnfiddle** [comment]
@janagombitova yes I saw that I already fixed it. @tangrammer we didn't have translations in the first place so I just made it like this as a quick fix. If there's no urgency then we can wait for loic etc to give translations

**Comment by finnfiddle** [comment]
@loicsans @tangrammer could you provide translations please for the following:
* Welcome
* Get started by adding datasets, visualisations and dashboards to your library from the top menu.
* Empty
* There are no items in this collection.
* No Results
* Please update your search and try again.

**Comment by tangrammer** [comment]
+ Welcome: Bienvenida
+ Comience añadiendo set de datos, visualizaciones, y tableros a su librería usando el menú superior: Get started by adding datasets, visualisations and dashboards to your library from the top menu.
+ Empty: Vacío
+ No hay resultados en esta colección:  There are no items in this collection.
+ Sin resultados: No Results
+ Por favor actualice su busqueda y pruebe otra vez: Please update your search and try again.

**Comment by finnfiddle** [comment]
translations task moved out into separate issue

https://github.com/akvo/akvo-lumen/issues/2119

**Pull request merged**